api/External.json

```diff
@@ -20,7 +20,7 @@
             "version_added": true
           },
           "ie": {
-            "version_added": false
+            "version_added": "≤6"
```

Comment:
What properties did the object have in IE6, given that the methods below are set to 7?

Reply:
I have no clue honestly, probably some proprietary IE-only implementation.  Though `window.external` _is_ defined in IE 6 (and as BCD states, all the way back to IE 4).

Reply:
Fair enough. It's not inconsistent, just curious since some of the data is missing (and shouldn't be added).